packages/status-js/package.json

```diff
@@ -50,5 +50,8 @@
   ],
   "publishConfig": {
     "access": "public"
+  },
+  "browser": {
```

Comment:
Field is not handled, see https://github.com/vitejs/vite/issues/7576#issuecomment-1152782131. Regardless, `crypto` now resolves to `var _crypto = {};` in the bundle.